R/pkg/R/functions.R

```diff
@@ -488,19 +488,35 @@ setMethod("initcap",
             column(jc)
           })
 
-#' isNaN
+#' isnan
 #'
-#' Return true iff the column is NaN.
+#' Return true if the column is NaN.
 #'
-#' @rdname isNaN
-#' @name isNaN
+#' @rdname isnan
+#' @name isnan
 #' @family normal_funcs
 #' @export
-#' @examples \dontrun{isNaN(df$c)}
-setMethod("isNaN",
+#' @examples \dontrun{isnan(df$c)}
+setMethod("isnan",
           signature(x = "Column"),
           function(x) {
-            jc <- callJStatic("org.apache.spark.sql.functions", "isNaN", x@jc)
+            jc <- callJStatic("org.apache.spark.sql.functions", "isnan", x@jc)
+            column(jc)
+          })
+
+#' isnull
+#'
+#' Return true if the column is NULL.
+#'
+#' @rdname isnull
+#' @name isnull
+#' @family normal_funcs
+#' @export
+#' @examples \dontrun{isnull(df$c)}
+setMethod("isnull",
```

Comment:
this gets tricky... R's usage for NULL is not the same as JVM's. In many cases it might be closer to R's NA though not exactly the same either.

eg. can't set a column in data.frame to NULL in the conventional way:
http://www.cookbook-r.com/Manipulating_data/Adding_and_removing_columns_from_a_data_frame/

Maybe it's fine to leave it as `isnull` for distinction - we really need some explanations on what is a NULL column.